### DISCUSSION TITLE
Organising (Dev) Conversations

### DISCUSSION THREAD

**Discussion by DanVanAtta** [triplea-game/triplea#10835] [General]
Would be great to update documentation on the best ways to communicate about the project.
Any thoughts if 'dev' topics should be discussed as 'github issues', 'github conversations', or 'forum threads'?

**Comment by RoiEXLab** [comment]
I think discussions about the code should be held on GitHub. Stuff like rule discussions (is behaviour A or B correct) is probably better on the forum because there are more people with knowledge about that there. And there are a lot of topic where the line becomes blurry
